src/utilities.css

```diff
@@ -0,0 +1,4 @@
+.welcome {
+  text-decoration: none;
+  color: #00000090;
+}
```

Comment:
Would this still work with what you are trying to do if you made it a 
```
const WelcomeLink = styled(Link)`
  text-decoration: none;
  color: #00000090;
`;
```
and changeds this file from .css to .jsx